### PR TITLE
Fixed some markdown stuff

### PR DESCRIPTION
**Pull request by apap04**
Made some things a bit clearer and followed markdown rules by shortening URLs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,17 +6,17 @@ We wanted to make it open source so everyone can get visibility into what we are
...
-- Run in terminal:
+- Run in terminal (Git Bash on Windows):
 	- `git submodule init`
 	- `git submodule update`
 	- `yarn`
 	- `./node_modules/.bin/gulp translations:compile`
 	- `yarn run dev`
 
-That should set up a tiny server that hosts the website for you on your computer at http://localhost:8080. Open that URL up in a web browser and you should have Game Jolt running!
+That should set up a tiny server that hosts the website for you on your computer at [localhost:8080](http://localhost:8080.) Open that URL up in a web browser and you should have Game Jolt running!
```
Comment (qwertychouskie) [inline README.md]: The period should be outside the parentheses.

Comment (apap04) [reply README.md]: right.

**Comment by apap04** [comment]
For some reason, commit [63f7b7f](https://github.com/gamejolt/gamejolt/pull/200/commits/63f7b7f61f433b5bad9f783b6b6dcfbd66cda943), messed up the file and formatted everything. It really didn't format everything, but messed it up.

**Comment by qwertychouskie** [comment]
There's still some unnecessary line breaks.

File: README.md [hunk 2]
```diff
@@ -2,21 +2,25 @@
 
 This is the whole frontend for Game Jolt. It powers the site and the client.
 
-We wanted to make it open source so everyone can get visibility into what we are working on. Browse the code to see how Game Jolt is put together. Feel free to offer suggestions on how to do things better, as well as contributing your own code. I'll get a better guide on how to contribute soon.
+We wanted to make it open source so everyone can get visibility into what we are working on. Browse
```
Comment (qwertychouskie) [inline README.md]: More line breaks here.

**Pull request closed without merging**